README.md

```diff
@@ -7,37 +7,47 @@ Rust FFI wrapper to libv4l.
 
 ## Build
 
+### Build by docker-compose (RECOMMENDED)
+
 ```sh
-cargo build
+(host)$ docker-compose build libv4l-build
+(host)$ docker-compose run --rm libv4l-build  # Just check if build succeeds
+
+# Manually invoke build command inside container
+(host)$ docker-compose run --rm libv4l-build bash
+(container)$ cargo build
+(container)$ file target/debug/liblibv4l_sys.rlib
+target/debug/liblibv4l_sys.rlib: current ar archive
 ```
 
-### Parameter
+### Native build (not RECOMMENDED)
```

Comment:
Why native build is not recommended?

Reply:
I felt it's a kind of troublesome to resolve dependencies by users themselves, rather better to use dockernized process.

But now I think users have freedom of choice so I will delete `RECOMMENDED` / `not RECOMMENDED`.

Reply:
3a822a8